clients/common/rocblas_gentest.py

```diff
@@ -196,13 +196,13 @@ def setdefaults(test):
     # These are only for dynamic defaults
     # TODO: This should be ideally moved to YAML file, with eval'd expressions.
 
-    if all([x in test for x in ('M', 'incx', 'strideScale')]) and test['function']=='ger_strided_batched':
+    if (test['function']=='ger_strided_batched' or test['function']=='scal_strided_batched') and all([x in test for x in ('M', 'incx', 'strideScale')]):
         test.setdefault('stride_x', int(test['M'] * abs(test['incx']) *
                                     test['strideScale']))
```

Comment:
I would reduce the line lengths to less than 80 (with small exceptions), and use `in` for list membership instead of Boolean expressions. I use `flake8` (a Python `lint`) to validate Python scripts.
```
  if (all([x in test for x in ('M', 'incx', 'strideScale')]):
      if test['function'] in ('ger_strided_batched', 'scal_strided_batched'):
          test.setdefault('stride_x', int(test['M'] * abs(test['incx']) *
                                      test['strideScale']))
``` 

Ideally the hierarchical structure of the `if` statements should reflect the structure of the program, and the same test should not be repeated twice in two different places.

Reply:
Addressed in 077ef48.